src/workspaces/common/state/useAppPolling.ts

```diff
@@ -51,7 +51,7 @@ export const useAppPolling = (name: string, namespace: string, workspace?: Works
       throw error;
     }
   };
-  const refreshApps = withErrorReporting('Error loading apps', loadApps) as (maybeStale?: boolean) => Promise<void>;
+  const refreshApps = withErrorReporting('Error loading apps')(loadApps);
```

Comment:
type cast no longer needed here because withErrorReporting (and its friends from error.ts) now have great type-flow.  The type signature of loadApps is properly flowed out to refreshApps const.

YAY!